frontend/src/app/main/ui/ds/colors.scss

```diff
@@ -15,6 +15,7 @@ $mint-250-10: #00d1b81a;
 $green-200: #a7e8d9;
 $green-500: #2d9f8f;
 $green-950: #0a2927;
+$green-1000: #0d1d1b;
```

Comment:
I can't find this token in the Foundations penpot file. Either let's get Design folks to add it, or use an existing green (like `green-950`).

Reply:
I forgot to change the values (because were defined afterwards) but the values are defined in the US, even if the DS is not updated yet:

![image](https://github.com/user-attachments/assets/3159a443-2288-458f-b684-c30021720c0a)